client/js/lounge.js

```diff
@@ -411,6 +422,18 @@ $(function() {
 		}
 
 		focus();
+
+		// Toggle userlist if its "openness" isn't what it should be (what is stored in localstorage)
+		const userlistShouldBeExpanded = storage.get("userlist-expanded") === "true"; // stored string -> boolean
+		if (userlistShouldBeExpanded !== utils.isUserlistVisible()) {
+			viewport.find(".chan.active .rt").click();
+		}
+
+		// Toggle networklist if its "openness" isn't what it should be (what is stored in localstorage)
```

Comment:
You don't really need to sync network list here (just on page load should be enough?)

Reply:
...right. I didn't put much thought into where I placed this one, was just thinking "put this next to this one" :+1: